addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/NetatmoHandlerFactory.java

```diff
@@ -46,7 +46,8 @@
  * @author Gaël L'hopital - Initial contribution
  */
 
-@Component(service = ThingHandlerFactory.class, immediate = true, configurationPid = "binding.netatmo")
+@Component(service = { ThingHandlerFactory.class,
+        NetatmoHandlerFactory.class }, immediate = true, configurationPid = "binding.netatmo")
```

Comment:
Maybe it is ok but I am surprised to find a list of 2 classes.

Reply:
Yes, you're right, no need to declare ThingHandlerFactory also